debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoUtils.java

```diff
@@ -220,6 +220,15 @@ public static ChangeStreamIterable<BsonDocument> openChangeStream(MongoClient cl
             return client.getDatabase(database).watch(pipeline.getStages(), BsonDocument.class);
         }
 
+        // capture scope is collection
+        if (config.getCaptureScope() == MongoDbConnectorConfig.CaptureScope.COLLECTION) {
+            var captureTarget = config.getCaptureTarget().orElseThrow();
+            var database = captureTarget.split("\\.")[0];
+            var collection = captureTarget.split("\\.")[1];
```

Comment:
Seeing as this gets repeated. I'd refactor `MongoDbConnectorConfig#getCaptureTarget` to return `Optional<CollectionId>` instead :)

Reply:
CollectionId actually requires both parts, so lets leave it like this for now.